nixos/modules/services/databases/arangodb.nix

```diff
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.arangodb;
+  stateDir = "/var/lib/arango";
```

Comment:
wouldn't make it sense to add an option for this (when I `grep` over nixpkgs, it seems as the services that require a `statePath` also provide an option for this)